tencentcloud/common/abstract_model.py

```diff
@@ -35,9 +35,9 @@ def dfs(obj):
     def _deserialize(self, params):
         return None
 
-    def to_json_string(self):
+    def to_json_string(self,json_indent=None):
```

Comment:
please use indent directly.
and your code style can be slightly improved by adding a space after the comma(,)